modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/EditObjectFieldContent.tsx

```diff
@@ -22,13 +22,25 @@ interface EditObjectFieldContentProps
 		| 'forbiddenChars'
 		| 'forbiddenLastChars'
 		| 'forbiddenNames'
+		| 'objectDefinitionExternalReferenceCode'
 		| 'objectFieldId'
 	> {
 	containerWrapper: ElementType;
+	dbObjectFieldRequired?: boolean;
 	errors: ObjectFieldErrors;
 	handleChange: React.ChangeEventHandler<HTMLInputElement>;
 	modelBuilder?: boolean;
+	objectDefinition: Pick<
+		ObjectDefinition,
+		| 'accountEntryRestricted'
+		| 'accountEntryRestrictedObjectFieldName'
+		| 'externalReferenceCode'
+		| 'modifiable'
+		| 'name'
+		| 'status'
+	>;
```

Comment:
Hey @igor-franca, as we discussed, to maintain code consistency, objectDefinition must be an `ObjectDefinition` or `Partial<ObjectDefinition>`, so in my proposal I removed all these interface declarations and replaced by `ObjectDefinition`;